### DISCUSSION TITLE
Manually add icons

### DISCUSSION THREAD

**Discussion by redneckeyeball** [ente-io/ente#4485] [Enhancements]
Many icons are missing. 
Twitter (I refuse to call it X)
Fritz!box
SNCB
No-ip

etc. I have the icons, but how can I add them ?
I have a github. Happy to make a commit if someone would be kind enough to point me in the right direction

**Comment by dnred** [comment]
[auth/docs/adding-icons.md](https://github.com/ente-io/ente/blob/main/mobile/apps/auth/docs/adding-icons.md)
